doorkeeper-orms.gemspec

```diff
@@ -10,17 +10,23 @@ Gem::Specification.new do |s|
   s.authors     = ["jasl"]
   s.email       = ["jasl9187@hotmail.com"]
   s.homepage    = "http://github.com/doorkeeper-gem/doorkeeper-orms"
-  s.summary     = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper."
-  s.description = "Doorkeeper with extracted ORM specifics, including mongoid 2-4 and mongo_mapper"
+  s.summary     = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
+  s.description = "Doorkeeper mongoid 2, 3, 4 and mongo_mapper ORMs"
```

Comment:
> How about just specifying the Doorkeeper repo with an ENV variable? As an ORM gem maintainer, I would have my own Doorkeeper repo which I could then check out any branch or SHA, then cd into the ORM gem's repo and run something like doorkeeper_src=../doorkeeper bundle exec rspec. As long as this is documented, I think the flexibility would outweigh the inconvenience.

This is not far enough from what we have now, but if it's not automated it just won't be reliable enough, I don't think.

> If we do a submodule, the submodule would have to be updated any time you want to test against a different version of doorkeeper, right?

I just pushed the submodule idea. I don't see this as a problem, as I do expect the ORM extensions and doorkeeper to go hand in hand with versioning, and to have quite strict versioning restrictions. Hopefully it's not needed, but I wouldn't be surprised if there's a minor release of minor versions almost per minor release of doorkeeper.

It doesn't need to be tested against each commit, but against the relevant tag (`v2.2.0` as of now).